src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-8130.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2022-11-15'
 version: 8.13.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.13.0.gem'
+features: ["Add support for Redis v5.0", "Add support for Sidekiq v7.0", "Update Rack v3.0 instrumentation to support the use of a block with Rack::Builder#run"]
```

Comment:
```suggestion
features: ["Add support for Redis 5.0", "Add support for Sidekiq 7.0", "Update Rack 3.0 instrumentation to support the use of a block with Rack::Builder#run"]
```

We're not putting `v` before version numbers in other entries, so let's drop them here too.